Source/Atj/data/mvp_scenario.json

```diff
@@ -452,6 +547,11 @@
                     "behavior": "move_to",
                     "target": "bulletin_board_1"
                 },
+                {
+                    "type": "execute_action",
+                    "sync_time": "0.01",
```

Comment:
Let's use a value of `0.1` instead of `0.01` here. I haven't tested against that smaller value. There's a bug where if the value is too small, the action gets skipped.